recipe.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Healthy Protein Pancakes</title>
+</head>
+<body>
+    <h2><Healthy Protein Pancake/h2>
```

Comment:
This should be the main header, which is a specific header number.

Reply:
solved